app/src/main/java/ar/scacchipa/twittercloneapp/data/repository/UserAccessTokenDataMapper.kt

```diff
@@ -1,4 +1,7 @@
-package ar.scacchipa.twittercloneapp.data
+package ar.scacchipa.twittercloneapp.data.repository
+
+import ar.scacchipa.twittercloneapp.data.IMapper
+import ar.scacchipa.twittercloneapp.data.datasource.UserAccessTokenData
 
 class UserAccessTokenDataMapper: IMapper<UserAccessTokenData, UserAccessTokenDomain> {
```

Comment:
this file should be at the data level not Repository

Reply:
inside data, you are missing like a model folder where you'll put all these kinds of file that are exactly that, the models used by the data layer

Reply:
A model folder inside the data folder was created and the POJO class was put in this.